examples/fram_i2c_simpletest.py

```diff
@@ -36,6 +36,6 @@
 ##       a buffer the size of slice, which may cause
 ##       problems on memory-constrained platforms.
 
-# values = list(range(100)) # or bytearray or tuple
-# fram[0] = values
-# print(fram[0:99])
+values = list(range(100))  # or bytearray or tuple
+fram[0:100] = values
```

Comment:
Lets keep the writes commented out in the example that way it's harder for someone to unknowingly cause wear on their device by running the example unmodified.